package.json

```diff
@@ -123,11 +125,5 @@
     "testRunner": "jest-jasmine2",
     "globalSetup": "./lib/puppeteer/setup.js",
     "globalTeardown": "./lib/puppeteer/teardown.js"
-  },
-  "husky": {
-    "hooks": {
-      "pre-commit": "npm run lint",
-      "pre-push": "npm test"
-    }
```

Comment:
these weren't being run, I think husky had been removed - possibly because the pre-push hook was blocking contributions from m2 macs where the results from VRT were different